pygen/pygen_src/isa/riscv_cov_instr.py

```diff
@@ -19,8 +19,11 @@
 import logging
 from enum import Enum, IntEnum, auto
 from bitstring import BitArray
-from pygen_src.target.rv32i import riscv_core_setting as rcs
-from pygen_src.riscv_instr_pkg import *
+from pygen.pygen_src.riscv_instr_pkg import *
```

Comment:
Hi @aneels3 and @saurabhsingh-pvips ,

A few importing-related comments here: 
1- In line 23, `cfg.argv...` is used but has not been imported yet (this is part of reported issue #714), I believe we need to import it first.
2- General tip here on the imports: we're appending "pygen/" in both of our current tests (`riscv_instr_base_test` as well as `riscv_instr_cov_test.py`). With that said, if we import from `pygen.pygen_src.whatever` in case someone runs it from terminal will raise an error saying that `pygen` is not recognized (as we're already within pygen directory). However, it works on most IDEs as they dynamically find and map directories as import statements happen. So to be consistent and avoid complicating the import statements, I'd suggest import directly from pygen_src.whatever (removing the pygen part in the beginning).